### ISSUE TITLE
EmulatedRokuServer listen_port parameter mandatory

### ISSUE THREAD

**Issue by clach04** [martonperei/emulated_roku#7]
Is there a reason why the listen_port EmulatedRokuServer is required? I've not used this tool (nor any Roku remotes/related software), but it seems reasonable to default this to 8060, which appears to be the Roku port used. It's above 1024, so no special permissions are needed on unix/linux platforms.

**Comment by martonperei** [comment]
Yes, with SSDP you can use any port, it doesn't have to be 8060, Roku remotes will find it nevertheless.  In theory it could find whatever free port it can, but I rather not set a default port.
Thanks for bringing this up!
